<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>D_essay</class>
 <widget class="QDialog" name="D_essay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文献详情</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>title:</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="l_title">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>booktitle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_booktitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>author:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tw1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>451</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>作者名</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>volume</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_volume">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>571</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>publication date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_data">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>journal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_journal">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_type">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="b1">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>570</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
